### PR TITLE
Contributors update

### PR DESCRIPTION
**Pull request by nkorinek**
Added a contributors file with the information that was previously stored in the README file.

### CODE REVIEW

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Contributors
+^^^^^^^^^^^^
+
+We've welcome any and all contributions. Below are some of the
+contributors to MatPlotCheck.
+
+.. image:: https://avatars.githubusercontent.com/u/43677611?size=120
+  :width: 60
+  :alt: Ryan Solvik
```
Comment (lwasser) [inline CONTRIBUTORS.rst]: oops Laroque (double check my spelling please)

File: README.md [hunk 2]
```diff
@@ -111,43 +111,6 @@ vt.assert_polygons(polygons_expected=polygons)
 Caveats: This repo likely misses edge cases of the many ways matplotlib plots can be created.
 Please feel free to submit bugs!
 
-
-## Active Contributors
```
Comment (lwasser) [inline README.md:115]: @nkorinek my apologies for not being clear. i want to keep the contributors in the readme so take what you did above and move it down here. i wanted to switch my name to the github image and then add your name below it with your github image as well.

File: README.md [hunk 3]
```diff
@@ -111,43 +111,6 @@ vt.assert_polygons(polygons_expected=polygons)
...
-
-- Leah Wasser
-
-## Dev Setup (to be moved to contributing)
```
Comment (lwasser) [inline README.md:119]: Then this part moves to contributing .rst as you did

File: README.md [hunk 4]
```diff
@@ -111,43 +111,6 @@ vt.assert_polygons(polygons_expected=polygons)
...
-
-Then setup all of the development requirements.
-
-```
-pip install -e .
-pip install -r dev-requirements.txt
-pre-commit install
-```
-
-## Contributors
```
Comment (lwasser) [inline README.md:136]: This part stays in the readme but let's fix ryan's name. i see this was my mistake!! can you help me fix it?

File: README.md [hunk 5]
```diff
@@ -111,43 +111,6 @@ vt.assert_polygons(polygons_expected=polygons)
...
-
-We've welcome any and all contributions. Below are some of the
-contributors to MatPlotCheck.
-
-<a title="Ryan Solvik" href="https://www.github.com/ryla5068"><img width="60" height="60" alt="Kylen Solvik" class="pull-left" src="https://avatars.githubusercontent.com/u/43677611?size=120" /></a>
-<a title="Kylen Solvik" href="https://www.github.com/ksolvik"><img width="60" height="60" alt="Kylen Solvik" class="pull-left" src="https://avatars.githubusercontent.com/u/24379590?size=120" /></a>
-<a title="Kristen Curry" href="https://www.github.com/kdcurry"><img width="60" height="60" alt="Kristen Curry" class="pull-left" src="https://avatars.githubusercontent.com/u/4032126?size=120" /></a>
-
-## How to Contribute
```
Comment (lwasser) [inline README.md:145]: This section how to contribute stays in the readme as well.

**Review by lwasser — CHANGES_REQUESTED** [review]

**Comment by nkorinek** [comment]
@lwasser ok I think I fixed it!

**Review by lwasser — APPROVED** [review]

**Comment by lwasser** [comment]
excellent! thank you @nkorinek merging this!

**Pull request merged**